includes/definitions/discovery/drac.yaml

```diff
@@ -20,6 +20,8 @@ modules:
                         - { value: 4, descr: nonCritical, graph: 1, generic: 1 }
                         - { value: 5, descr: critical, graph: 1, generic: 2 }
                         - { value: 6, descr: nonRecoverable, graph: 1, generic: 2 }
+                    high_limit: 5 
```

Comment:
high_limit, warn_limit and other limit values are not valid for state sensors.   Generic values should be set instead.

Reply:
But they work :). And if I define warning values manually if have to enter these values as well.

![image](https://user-images.githubusercontent.com/53938045/148056907-87490d4e-185d-45af-8fa1-f5bb95f4a1a0.png)
There is no way that I define that a specific predefined value should trigger a warning event.

Maybe there is a general logic problem with the static values?